src/chapters/forms/checkbox-input.rst

```diff
@@ -0,0 +1,84 @@
+Checkbox Input
+==============
+
+A checkbox input represents a box to check. Checkbox inputs can be
+used by themselves or in groups. Checkbox inputs are best used with ``<label>`` tags.
+
+.. role:: raw-html(raw)
+   :format: html
+
+.. list-table::
+   :header-rows: 1
+
+   * - Type
+     - Syntax
+     - Description
+     - Demo
+   * - checkbox
+     - ``<input type="checkbox" name="signUp"/>``
+     - A small box for marking form option as *checked*.
+     - :raw-html:`<label>sign up<input type="checkbox" name="signUp"/></label>`
+
+Examples
+--------
+
+.. admonition:: Example
+
+   One checkbox. No ``value`` attribute is set, so the default value of ``on`` is submitted.
+
+   .. sourcecode:: html
+
+       <label>crew<input type="checkbox" name="crewReady"/></label>
+
+   **Submitted** (if checked)
+
+   ::
+
+      crewReady=on
+
+   `Run it at repl.it <https://repl.it/@launchcode/checkbox-inputs-example>`__
+
+.. admonition:: Example
+
+   Multiple checkbox inputs. All with *different* ``name`` attributes.
+
```

Comment:
Just thinking maybe a little more info or a Question about what is the difference between same name vs different?